x/emissions/types/allora_type_aliases.go

```diff
@@ -16,3 +18,11 @@ type TaskReward struct {
 	TopicId TopicId
 	Type    TaskRewardType
 }
+
+// contains a score and a pointer to the
+// actor with the next lowest score,
+// for updating the lowest scores in the keeper
+type ScoreAndNextLowest struct {
+	Score      Score
+	NextLowest string
```

Comment:
There a linked list somewhere I'm missing...?

Reply:
Good catch, I started to write a linked list and then decided to not 

This is garbage leftover code